wp-content/themes/engage/src/Models/Homepage.php

```diff
@@ -71,37 +80,17 @@ public function getMoreRecentResearch($featuredSliderPosts, $verticalName){
 			"social-platforms" => 2,
             "science-communication" => 2
         ];
-        $allRecentResearch = $this->queryPosts(false, $verticalName);
-        $moreRecentResearch = array();
-        for($i = 0; $i<count($allRecentResearch) && $numFeaturedPerVertical[$verticalName] > 0; $i++){
-            $curr = $allRecentResearch[$i];
-            // make sure not already displaying the post, and that we haven't used up the allotted num of posts
-            if($this->notInSlider($curr, $featuredSliderPosts)){
-                $numFeaturedPerVertical[$verticalName]--;
-                array_push($moreRecentResearch, $curr);
-            }
-        }
-        return $moreRecentResearch;
+        $allRecentResearch = $this->queryPosts(false, $verticalName, $numFeaturedPerVertical[$verticalName]);
+        return $allRecentResearch;
 	}
 
-    // check if a post with the same id as curr is not on the slider
-    public function notInSlider($curr, $featuredSliderPosts){
-		foreach($featuredSliderPosts as $featurePost){
-			// posts with same id are the same post
-            if($curr->id == $featurePost->id){
-                return false;
-            }
-		}
-        return true;
-    }
-
     // query the posts with the given arguments
-    public function queryPosts($is_featured, $verticalName){
-		$posts = ($is_featured ? '1' : '4'); // 4 featured posts (1 for each vertical), 4 otherwise to account for if already used up in featured posts (and want to display more than 2)
+    public function queryPosts($is_featured, $verticalName, $numberOfPosts){
         $args = [
             'postType' => 'research',
             'vertical' => $verticalName,
-			 'postsPerPage' => $posts
+			 'postsPerPage' => $numberOfPosts,
+			 'post__not_in' => array_map(function($post){return $post->id;}, $this->recent)
```

Comment:
Rather than add `post__not_in`, you can use the `'extraQuery' 	=> []` option so you won't need your next commit of modifying the `getRecentPosts`.

```
'extraQuery' => ['post__not_in' => ... ],
```

It's not the best framework, but basically `extraQuery` is just a place to put whatever WP_Query options you want and it will merge it all together for you later.